app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>List of Contestants</h1>
+
+<% @contestants.each do |contestant| %>
+  <section id="contestant-<%= contestant.id %>">
+      <p><%= contestant.name %></p>
+      <% project_list = ContestantProject.all.select {|cp| cp.contestant_id == contestant.id} %>
```

Comment:
Instead of doing this, you may also want to look into leveraging how you can access your associations directly. Maybe something along the lines of `contestant.projects` by chance.